scripts/bert/finetune_squad.py

```diff
@@ -0,0 +1,459 @@
+                    default=0.1,
+                    help='ratio of warmup steps used in NOAM\'s stepsize schedule. default is 0.1')
+
+parser.add_argument('--log_interval',
+                    type=int,
+                    default=50,
+                    help='report interval. default is 50')
+
+parser.add_argument('--max_seq_length',
+                    type=int,
+                    default=384,
+                    help='The maximum total input sequence length after WordPiece tokenization.'
+                    'Sequences longer than this will be truncated, and sequences shorter '
+                    'than this will be padded. default is 384')
+
+parser.add_argument('--doc_stride',
+                    type=int,
+                    default=128,
+                    help='When splitting up a long document into chunks, how much stride to '
+                    'take between chunks. default is 128')
+
+parser.add_argument('--max_query_length',
+                    type=int,
+                    default=64,
+                    help='The maximum number of tokens for the question. Questions longer than '
+                    'this will be truncated to this length. default is 64')
+
+parser.add_argument('--n_best_size',
+                    type=int,
+                    default=20,
+                    help='The total number of n-best predictions to generate in the '
+                    'nbest_predictions.json output file. default is 20')
+
+parser.add_argument('--max_answer_length',
+                    type=int,
+                    default=30,
+                    help='The maximum length of an answer that can be generated. This is needed '
+                    'because the start and end predictions are not conditioned on one another.'
+                    ' default is 30')
+
+parser.add_argument('--version_2',
+                    action='store_true',
+                    help='SQuAD examples whether contain some that do not have an answer.')
+
+parser.add_argument('--null_score_diff_threshold',
+                    type=float,
+                    default=-2.0,
+                    help='If null_score - best_non_null is greater than the threshold predict null.'
+                    ' default is -2.0')
+
+parser.add_argument('--gpu',
+                    action='store_true',
+                    help='whether to use gpu for finetuning')
+
+args = parser.parse_args()
+
+output_dir = args.output_dir
+if not os.path.exists(output_dir):
+    os.mkdir(output_dir)
+
+fh = logging.FileHandler(os.path.join(
+    args.output_dir, 'finetune_squad.log'), mode='w')
+fh.setLevel(logging.INFO)
+fh.setFormatter(formatter)
+console = logging.StreamHandler()
+console.setLevel(logging.INFO)
+console.setFormatter(formatter)
+log.addHandler(console)
+log.addHandler(fh)
+
+log.info(args)
+
+model_name = args.model
+dataset_name = args.dataset_name
+only_predict = args.only_predict
+model_parameters = args.model_parameters
+lower = args.cased
```

Comment:
```suggestion
lower = args.uncased
```

This seems like a bug.